src/SUMMARY.md

```diff
@@ -35,17 +35,18 @@
 
 # Contributing to Rust
 
-- [Introduction](./contributing.md)
+- [Contribution Procedures](./contributing.md)
```

Comment:
Hmm, what do you think about keeping the original name? This section ("Contributor to Rust") has a *ton* of chapters, and it seems a bit overwhelming without an introduction chapter.

Reply:
Yes, exactly. I'm not sure if anywhere links to this page right now, but if it does we should change it to point to Getting Started instead.

Reply:
FYI:

```console
❯ rg -F -C1 contributing.md
src/getting-started.md
152-[iceb]: ./notification-groups/cleanup-crew.md
153:[wd]: ./contributing.md#writing-documentation
154-[wg]: https://rust-lang.github.io/compiler-team/working-groups/

src/SUMMARY.md
37-
38:- [Introduction](./contributing.md)
39-- [About the compiler team](./compiler-team.md)

src/about-this-guide.md
26-[p1]: ./building/how-to-build-and-run.html
27:[p2]: ./contributing.md
28-[p3]: ./part-2-intro.md
--
50-
51:[subsection on writing documentation in this guide]: contributing.md#contributing-to-rustc-dev-guide
52-
```

Reply:
ok, done :)